apps/emqx_management/test/emqx_mgmt_api_SUITE.erl

```diff
@@ -101,6 +173,34 @@ t_cluster_query(_Config) ->
     end,
     ok.
 
+t_paging(_) ->
+    emqx_mgmt_api_test_util:init_suite(),
+    try
+        Path0 = emqx_mgmt_api_test_util:api_path(["banned?page=1"]),
+        {ok, _} = emqx_mgmt_api_test_util:request_api(get, Path0),
+        Path1 = emqx_mgmt_api_test_util:api_path(["banned?page=10"]),
+        {ok, _} = emqx_mgmt_api_test_util:request_api(get, Path1)
+    after
+        emqx_mgmt_api_test_util:end_suite()
+    end.
```

Comment:
```suggestion
    Path0 = emqx_mgmt_api_test_util:api_path(["banned?page=1"]),
    {ok, _} = emqx_mgmt_api_test_util:request_api(get, Path0),
    Path1 = emqx_mgmt_api_test_util:api_path(["banned?page=10"]),
    {ok, _} = emqx_mgmt_api_test_util:request_api(get, Path1).
```

?

Reply:
Also, should the resulting details be checked?  Or maybe that's already well covered by `t_cluster_query`?

Reply:
I need to call `init_suite/0` since I want to talk to the actual API. Alternatively I could call `emqx_mgmt_api_clients:clients/2`.

Reply:
Maybe we could move `{init,end}_suite/0` to `{init,end}_per_suite/1`, so it runs once for the whole suite?